src/_posts/databases/postgresql/2000-01-01-accessing.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Accessing Your Scalingo for PostgreSQL® Addon
+nav: Accessing
+modified_at: 2023-12-01 00:00:00
+tags: databases postgresql addon
+index: 6
+---
+
+
+Remotely accessing your Scalingo for PostgreSQL® database can sometimes be
+useful, for example, to conduct investigations, to check or compute data
+locally, to dump the database content,...
+
+We provide several ways to access your database remotely. Chosing one mostly
+depends on your needs and preferences.
+
+## Using the Interactive Remote Console
+
+1. Make sure you have correctly [setup the Scalingo Command Line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, open a console for your Scalingo for PostgreSQL®
+   addon:
+   ```bash
+   scalingo --app my-app pgsql-console
+   ```
+   The output should look like this:
+   ```bash
+   -----> Starting container one-off-7872  Done in 0.479 seconds
+   -----> Connecting to container [one-off-7872]...  
+   -----> Process 'pgsql-console my_app_4553' is starting...  
+
+   Hello from bashrc.
```

Comment:
issue: I don't think this is supposed to be in the output. That may be specific to your test app.
```suggestion
```